packages/ag-charts-community/src/chart/toolbar/toolbarTypes.ts

```diff
@@ -45,3 +45,9 @@ export interface ToolbarButton extends ToolbarButtonConfig {
     role?: 'button' | 'switch';
     checkedOverrides?: ToolbarButtonConfig;
 }
+
+export interface Anchor {
```

Comment:
To be more specific:

```suggestion
export interface ToolbarAnchor {
```